From 844f61069896fe3f549ab425d731c061028f697c Mon Sep 17 00:00:00 2001
From: Adrian Bunk <bunk@stusta.de>
Date: Tue, 12 May 2020 09:44:05 +0300
Subject: src: Fix build with gcc 10

gcc 10 enables -fno-common by default which fails without these typedefs

Signed-off-by: Adrian Bunk <bunk@stusta.de>
Signed-off-by: Richard Purdie <richard.purdie@linuxfoundation.org>

[Retrieved from:
http://git.yoctoproject.org/cgit/cgit.cgi/matchbox-window-manager/commit/?id=844f61069896fe3f549ab425d731c061028f697c]
Signed-off-by: Fabrice Fontaine <fontaine.fabrice@gmail.com>
---
 src/mbtheme.h | 2 +-
 src/structs.h | 2 +-
 2 files changed, 2 insertions(+), 2 deletions(-)

diff --git a/src/mbtheme.h b/src/mbtheme.h
index aa9a7c5..ad03bde 100644
--- a/src/mbtheme.h
+++ b/src/mbtheme.h
@@ -46,7 +46,7 @@ typedef struct _mb_theme_param
 
 } MBThemeParam;
 
-enum {
+typedef enum {
   LAYER_GRADIENT_HORIZ = 1,
   LAYER_GRADIENT_VERT,
   LAYER_LABEL,
diff --git a/src/structs.h b/src/structs.h
index 24985e7..8f53e72 100644
--- a/src/structs.h
+++ b/src/structs.h
@@ -148,7 +148,7 @@
 
 /* Atoms, if you change these check ewmh_init() first */
 
-enum {
+typedef enum {
   WM_STATE = 0,
   WM_CHANGE_STATE,
   WM_PROTOCOLS,
-- 
cgit v1.2.2-1-g5e49

